source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -68,7 +68,7 @@ To remove a library dependency from a project, select :guilabel:`Manage Current
 Libraries
 ---------
 
-.. warning:: Many vendors, such as CTRE & REV, recommend using their offline installers over using the online vendor json URL.
+.. warning:: Many vendors, such as CTRE & REV, recommend using their offline installers over using the online vendor json URL.  This recommendation is because the offline installer is bundled with the additional programs that are extremely useful when working with their devices.
```

Comment:
GitHub UI is being weird. But this sentence "This recommendation is because the offline installer is bundled with the additional programs that are extremely useful when working with their devices." should be changed to "This recommendation is because the offline installer is bundled with additional programs that are extremely useful when working with their devices."